### PR TITLE
More faqs

### LINKED ISSUE #76: Operation/troubleshooting documentation
In addition to getting start documentation, we need to some operation docs to guide users to operate in the production cluster.

- [x] Deleting/removing the underlying volume: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#deletingremoving-the-underlying-volume
  - https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#managing-your-local-volumes
- [x] deployment issue when docker --init flag is enabled (https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/50)
- [x] ways to prepare disks in production
  - https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner#managing-your-local-volumes
- [x] How to fix when a PV (reclaimPolicy=Delete) is released but not able to be deleted
- [x] do not delete PV to recycle volume
  - provisioner has no reliable way to know whether volume without a matching PV has been used before, so it will not clean up it
- [x] How to upgrade provisioner configuration (related to https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/12)
- [x] How to migrate volumes to another storage class (depends on https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/75)

### PR DESCRIPTION
**Pull request by cofyc**
fixes #76

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -49,3 +53,74 @@ Workarounds before the fix is released:
...
+So provisioner will always discover volume as a new PV if no existing PV is
+associated with it. The volume data will not be cleaned in this phrase, and old
+data in it may leak other applications.
+
+So you must not delete PV objects by yourself, always delete PVC objects and
+set `PersistentVolumeReclaimPolicy` to `Delete` to clean volume data of
+associated PVs.
+
+## PV with delete reclaimPolicy is released but not going to be reclaimed
+
```
Comment (cofyc) [inline docs/faqs.md]: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/issues/6

File: docs/faqs.md [hunk 2]
```diff
@@ -49,3 +53,74 @@ Workarounds before the fix is released:
...
+  hardware failures, please fix the volume. If the volume can be recovered,
+  provisioner will continue to clean the volume data and reclaim the PV object.
+  If the volume cannot be recovered, you can remove the volume (or disk) from
+  the node and delete the PV object manually.
+
+Of course, on a specific platform if you have a reliable mechanism to detect if
+a volume is permanently deleted or cannot recover, you can write an operator or
+sidecar to automate this.
+
+## Why my application uses an empty volume when node gets recreated in GCP
```
Comment (cofyc) [inline docs/faqs.md]: https://github.com/pingcap/tidb-operator/issues/408#issuecomment-485351029

File: docs/faqs.md [hunk 3]
```diff
@@ -49,3 +53,74 @@ Workarounds before the fix is released:
...
+
+For example, in GKE when nodes get recreated, local SSDs will be recreated and
+mounted at the same paths (note: `--keep-disks` cannot be used to keep local
+SSDs because `autoDelete` cannot be set to false on local SSDs). Pods using old
+volumes will start with empty volumes because paths of PV objects will get
+mounted with newly created disks.
+
+If your application does not expect this behavior, you should use
+[`--local-ssd-volumes`](https://cloud.google.com/sdk/gcloud/reference/alpha/container/node-pools/create)
+and configure provisioner to discover volumes under `/mnt/disks/by-uuid/google-local-ssds-scsi-fs` or
```
Comment (msau42) [inline docs/faqs.md]: This example contains that config: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/helm/generated_examples/gce.yaml

Comment (cofyc) [reply docs/faqs.md]: A link to this example is added.

**Pull request merged**